wallet/src/de/schildbach/wallet/ui/invite/InviteHandler.kt

```diff
@@ -108,6 +111,11 @@ class InviteHandler(val activity: AppCompatActivity) {
         }
     }
 
+    private fun getMainTask(): ActivityManager.AppTask {
+        val activityManager = activity.getSystemService(AppCompatActivity.ACTIVITY_SERVICE) as ActivityManager
+        return activityManager.appTasks.last()
+    }
+
```

Comment:
On Android 11, we get this crash if the app is not running before we click an invite.

If the app is running, then this exception doesn't happen and onboarding with invitations will proceed.  

On Android 10, the we don't get this exception, but  the result is the same.  Nothing happens after the Verify Invite dialog disappears.
```
2021-08-13 11:44:30.857 25248-25248/org.dash.dashpay.testnet E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.dash.dashpay.testnet, PID: 25248
    java.lang.NullPointerException: Attempt to invoke virtual method 'void com.android.server.wm.DisplayContent.layoutAndAssignWindowLayersIfNeeded()' on a null object reference
        at android.os.Parcel.createExceptionOrNull(Parcel.java:2379)
        at android.os.Parcel.createException(Parcel.java:2357)
        at android.os.Parcel.readException(Parcel.java:2340)
        at android.os.Parcel.readException(Parcel.java:2282)
        at android.app.IAppTask$Stub$Proxy.startActivity(IAppTask.java:291)
        at android.app.Instrumentation.execStartActivityFromAppTask(Instrumentation.java:2022)
        at android.app.ActivityManager$AppTask.startActivity(ActivityManager.java:4857)
        at de.schildbach.wallet.ui.invite.InviteHandler.handle(InviteHandler.kt:123)
        at de.schildbach.wallet.ui.InviteHandlerActivity$initViewModel$2.onChanged(InviteHandlerActivity.kt:94)
        at de.schildbach.wallet.ui.InviteHandlerActivity$initViewModel$2.onChanged(InviteHandlerActivity.kt:29)
        at androidx.lifecycle.LiveData.considerNotify(LiveData.java:131)
        at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:149)
        at androidx.lifecycle.LiveData.setValue(LiveData.java:307)
        at androidx.lifecycle.MutableLiveData.setValue(MutableLiveData.java:50)
        at androidx.lifecycle.LiveData$1.run(LiveData.java:91)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
     Caused by: android.os.RemoteException: Remote stack trace:
        at com.android.server.wm.ActivityStack.onChildPositionChanged(ActivityStack.java:3187)
        at com.android.server.wm.WindowContainer.onChildAdded(WindowContainer.java:547)
        at com.android.server.wm.WindowContainer.setParent(WindowContainer.java:363)
        at com.android.server.wm.WindowContainer.addChild(WindowContainer.java:537)
        at com.android.server.wm.Task.addChild(Task.java:1370)

```